src/main/models/subscription.ts

```diff
@@ -0,0 +1,6 @@
+export interface Subscription {
+  userId: number;
+  userEmail: string;
```

Comment:
I got this from the mock file used userSubscriptions.json

Reply:
That is a mock file that was created in October, whereas i sent you an updated schema that is how it will look yesterday, please make sure youre using up to date data 😊

Reply:
Where did you send it? I can't find it

Reply:
properties removed